packages/playground/windows/playground-composition/packages.config

```diff
@@ -10,4 +10,5 @@
   <package id="Microsoft.VCRTForwarders.140" version="1.0.2-rc" targetFramework="native"/>
   <package id="Microsoft.Windows.CppWinRT" version="2.0.210312.4" targetFramework="native"/>
   <package id="Microsoft.JavaScript.Hermes" version="0.1.15" targetFramework="native"/>
+  <package id="Microsoft.WindowsAppSDK" version="1.4.230913002" targetFramework="native" />
```

Comment:
Do we even need packages.config anymore? Can we just put this as a PackageReference (esp in the new external property sheets / targets)?